pytest_splunk_addon/plugin.py

```diff
@@ -116,6 +120,62 @@ def pytest_sessionstart(session):
         sample_generator.get_samples(store_events)
 
 
+def pytest_sessionfinish(session, exitstatus):
+    try:
+        with open(f".{os.sep}splunk_type.txt") as splunk_type_file:
+            splunk_data = [line.rstrip() for line in splunk_type_file]
+            splunk_type = splunk_data[0]
+            splunk_keep_alive = splunk_data[1]
+            splunk_app_path = splunk_data[2]
+        if (
+            (os.environ.get("PYTEST_XDIST_WORKER") == None)
+            and (splunk_type == "kubernetes")
```

Comment:
You don't need parentheses here.

`and splunk_type == "kubernetes"` should work.